docs/reference/type-system-features.md

```diff
@@ -228,18 +317,65 @@ This page summarizes the support for various type system features in ty.
+| `reveal_type(value)`                               | ✅                                                    |
+| `TYPE_CHECKING` constant                           | ✅                                                    |
+| `no_type_check` decorator                          | ✅                                                    |
+| `type: ignore` comments                            | ✅                                                    |
+| `@deprecated` decorator                            | ✅                                                    |
+| `@override` decorator                              | ✅                                                    |
+| Diagnostic: override without `@override` decorator | ❌ [#155](https://github.com/astral-sh/ty/issues/155) |
+| Redundant `cast` diagnostic                        | ✅                                                    |
+
+## Module resolution
+
+[Official documentation](https://typing.python.org/en/latest/spec/distributing.html)
+
+| Feature                                           | Status                                                  |
+| ------------------------------------------------- | ------------------------------------------------------- |
+| Stub files (`.pyi`)                               | ✅                                                      |
+| Stub packages (`<package>-stubs`)                 | ✅                                                      |
+| Partial stub packages (`py.typed` with `partial`) | ✅                                                      |
+| Namespace packages (PEP 420)                      | ✅                                                      |
+| Relative imports                                  | ✅                                                      |
+| `__all__` declarations                            | ✅                                                      |
+| `__all__` mutations (`.append`, `.extend`, `+=`)  | ✅                                                      |
+| `__all__` with submodule `__all__`                | ✅                                                      |
+| Wildcard (`*`) imports                            | ✅                                                      |
+| Wildcard imports in stubs re-export               | ✅                                                      |
+| Re-export conventions (`import X as X`)           | ✅                                                      |
+| Conditional re-exports in stub files              | ✅                                                      |
+| Reachability constraints in imports               | ✅                                                      |
+| Cyclic imports                                    | ✅                                                      |
+| `py.typed` marker files                           | ⚠️ ignored for type checking                            |
```

Comment:
I think we respect these as much as we ever intend to? I'd be inclined to check this one off